samples/bookinfo/platform/kube/istio-rbac-enable.yaml

```diff
@@ -1,41 +0,0 @@
-apiVersion: "config.istio.io/v1alpha2"
-kind: authorization
```

Comment:
why are you removing examples? isn't it referred in the docs?

Reply:
These are old samples for mixer based rbac, the docs now referrers to the new samples in `samples/bookinfo/platform/kube/rbac`